erpnext/accounts/doctype/account/chart_of_accounts/chart_of_accounts.py

```diff
@@ -75,15 +81,6 @@ def add_suffix_if_duplicate(account_name, account_number, accounts):
 
 	return account_name, account_name_in_db
 
-def identify_is_group(child):
```

Comment:
We could keep this and still be able to set currency right? Any particular reason to remove auto-setting of `is_group`?

Since `create_charts()` is used in tally migrator where we just pass the nested accounts dictionary without setting `is_group` and the above function sets it correctly.

Reply:
The main reason for removing group is that it tends to assume the group of an account based on the passed keys, which seems both inaccurate and an inefficient way of assigning groups. Instead, I'm suggesting a manual but sure way of setting it.

For Tally, however, we could look into restoring this function only, and then try and phase that into a more accurate flow as well.

What do you think?